src/labels/CustomLabels.labels

```diff
@@ -7403,6 +7443,30 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>healthLabelContactData</shortDescription>
         <value>Contact Data</value>
     </labels>
+    <labels>
+        <fullName>healthLabelDataBatchNotRun</fullName>
+        <categories>Health Check</categories>
+        <language>en_US</language>
+        <protected>false</protected>
```

Comment:
Protected needs to be true. There are two others that need fixing as well.

Reply:
@force2b Thanks for the review. The code is updated and is ready for review again.